chall01/osalmine.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python3
+
+import sys
+
+morse = { 'A':'.-', 'B':'-...',
+                    'C':'-.-.', 'D':'-..', 'E':'.',
+                    'F':'..-.', 'G':'--.', 'H':'....',
+                    'I':'..', 'J':'.---', 'K':'-.-',
+                    'L':'.-..', 'M':'--', 'N':'-.',
+                    'O':'---', 'P':'.--.', 'Q':'--.-',
+                    'R':'.-.', 'S':'...', 'T':'-',
+                    'U':'..-', 'V':'...-', 'W':'.--',
+                    'X':'-..-', 'Y':'-.--', 'Z':'--..'}
```

Comment:
That's I approve! Smart! 
One little suggestion (in case you hardcoded it), you could just use codes given and make dict out of them using python like that: 
```
morse = '.- -... -.-. -.. . ..-. --. .... .. .--- -.- .-.. -- -. --- .--. --.- .-. ... - ..- ...- .-- -..- -.-- --..'.split(' ')
morse = {chr(65 + i) : morse[i] for i in range(len(morse))} # 65 is code of 'A'
```
But that's not all! You can also add a space as a key to your dictionary instead of checking it in the printing loop:
`morse[" "] = " "`
and that's it :)